libs/features/georoutingv2/georoutingv2.js

```diff
@@ -291,12 +304,21 @@ export default async function loadGeoRouting(conf, createTagFunc, getMetadataFun
   if (!urlGeoData) return;
 
   if (storedLocale || storedLocale === '') {
+    const urlLocaleGeo = urlLocale.split('_')[0];
+    const storedLocaleGeo = storedLocale.split('_')[0];
     // Show modal when url and cookie disagree
-    if (urlLocale.split('_')[0] !== storedLocale.split('_')[0]) {
-      const localeMatches = json.georouting.data.filter((d) => d.prefix === storedLocale);
+    if (urlLocaleGeo !== storedLocaleGeo) {
+      const localeMatches = json.georouting.data.filter(
+        (d) => d.prefix === storedLocale,
+      );
       const details = await getDetails(urlGeoData, localeMatches, json.geos.data);
       if (details) {
         await showModal(details);
+        const modUrlLocaleGeo = urlLocaleGeo || 'us';
+        const modStoredLocaleGeo = storedLocaleGeo || 'us';
+        sendAnalytics(
+          new Event(`Load:${modUrlLocaleGeo}-${modStoredLocaleGeo}|Geo_Routing_Modal`),
```

Comment:
The variables introduce more confusion than clarity IMO, so I'd just inline them
```suggestion
        sendAnalytics(
          new Event(`Load:${urlLocaleGeo || 'us'}-${storedLocaleGeo || 'us'}|Geo_Routing_Modal`),
```

Reply:
This is done.